app/Http/Controllers/Api/UserBiosController.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace Symposium\Http\Controllers\Api;
+
+use Illuminate\Support\Facades\App;
+use Illuminate\Support\Facades\Auth;
+use LucaDegasperi\OAuth2Server\Facades\AuthorizerFacade as Authorizer;
+use Symposium\ApiResources\Bio;
+use User;
+
+class UserBiosController extends BaseController
+{
+    /**
+     * Display all of the given user's bios
+     *
+     * @return Response
+     */
+    public function index($userId)
+    {
+        if ($userId != Authorizer::getResourceOwnerId()) {
+            App::abort(404);
+        }
+
+        $bios = User::find($userId)->bios;
+
+        $return = [];
+
+        foreach ($bios as $bio) {
+            $resource = new Bio($bio);
+
+            $return[] = [
+                'id' => $resource->getId(),
+                'type' => $resource->getType(),
+                'attributes' => $resource->attributes()
+            ];
+        }
```

Comment:
Should do a map here then you don't need to give `$return` a default value.

Also, if you make `Bio` implement `Jsonnable` or whatever, I'm pretty sure it will get auto-cast by Laravel on the way out.

So you could just do this as the whole deal:

```
return response()->jsonApi([
    'data' => $bios->map(function ($bio) {
        return new Bio($bio);
    }),
]);
```